fury/ui/tests/test_containers.py

```diff
@@ -250,113 +250,118 @@ def test_ui_image_container_2d(interactive=False):
-        tab_ui.update_element(3, ui.TextBlock2D(), (0.5, 0.5, 0.5))
-
-    npt.assert_equal('Tab 1', tab_ui.tabs[0].title)
-    npt.assert_equal('Tab 2', tab_ui.tabs[1].title)
-    npt.assert_equal('Tab 3', tab_ui.tabs[2].title)
-
-    npt.assert_equal(3, tab_ui.nb_tabs)
-
-    collapses = itertools.count()
-    changes = itertools.count()
-
-    def collapse(tab_ui):
-        if tab_ui.collapsed:
-            next(collapses)
-
-    def tab_change(tab_ui):
-        next(changes)
-
-    tab_ui.on_change = tab_change
-    tab_ui.on_collapse = collapse
+        tab_ui = ui.TabUI(
+            position=(50, 50), size=(300, 300), nb_tabs=3, draggable=True,
+            tabBar_pos=i)
 
-    event_counter = EventCounter()
-    event_counter.monitor(tab_ui)
+        tab_ui.tabs[0].title = 'Tab 1'
+        tab_ui.tabs[1].title = 'Tab 2'
+        tab_ui.tabs[2].title = 'Tab 3'
 
-    current_size = (800, 800)
-    show_manager = window.ShowManager(size=current_size, title='Tab UI Test')
-    show_manager.scene.add(tab_ui)
+        npt.assert_equal(tab_ui.tabs[0].title_bold, False)
+        npt.assert_equal(tab_ui.tabs[1].title_bold, False)
+        npt.assert_equal(tab_ui.tabs[2].title_bold, False)
 
-    if interactive:
-        show_manager.record_events_to_file(recording_filename)
-        print(list(event_counter.events_counts.items()))
-        event_counter.save(expected_events_counts_filename)
-    else:
-        show_manager.play_events_from_file(recording_filename)
-        expected = EventCounter.load(expected_events_counts_filename)
-        event_counter.check_counts(expected)
+        tab_ui.tabs[0].title_bold = True
+        tab_ui.tabs[1].title_bold = False
+        tab_ui.tabs[2].title_bold = True
+
+        npt.assert_equal(tab_ui.tabs[0].title_bold, True)
+        npt.assert_equal(tab_ui.tabs[1].title_bold, False)
+        npt.assert_equal(tab_ui.tabs[2].title_bold, True)
+
+        npt.assert_equal(tab_ui.tabs[0].title_color, (.0, .0, .0))
+        npt.assert_equal(tab_ui.tabs[1].title_color, (.0, .0, .0))
+        npt.assert_equal(tab_ui.tabs[2].title_color, (.0, .0, .0))
+
+        tab_ui.tabs[0].title_color = (1, 0, 0)
+        tab_ui.tabs[1].title_color = (0, 1, 0)
+        tab_ui.tabs[2].title_color = (0, 0, 1)
+
+        npt.assert_equal(tab_ui.tabs[0].title_color, (1., .0, .0))
+        npt.assert_equal(tab_ui.tabs[1].title_color, (.0, 1., .0))
+        npt.assert_equal(tab_ui.tabs[2].title_color, (.0, .0, 1.))
+
+        npt.assert_equal(tab_ui.tabs[0].title_font_size, 18)
+        npt.assert_equal(tab_ui.tabs[1].title_font_size, 18)
+        npt.assert_equal(tab_ui.tabs[2].title_font_size, 18)
+
+        tab_ui.tabs[0].title_font_size = 10
+        tab_ui.tabs[1].title_font_size = 20
+        tab_ui.tabs[2].title_font_size = 30
+
+        npt.assert_equal(tab_ui.tabs[0].title_font_size, 10)
+        npt.assert_equal(tab_ui.tabs[1].title_font_size, 20)
+        npt.assert_equal(tab_ui.tabs[2].title_font_size, 30)
```

Comment:
These tests are not required again. Update the unit test to check the functionality you have added as this is repetition of the tests which are already being checked.